### ISSUE TITLE
Target models bug

### ISSUE THREAD

**Issue by Baelroak** [itsHyper/qb-rentals#7]
Using target models for qb-target is bad idea, it makes it so any ped that is spawned in using that ped hash will have the rental options, for example I noticed this when a random ped at the hospital let me rent a car, but it teleported me to the rental location.

["VehicleRental"] = {
        models = {
            `a_m_y_business_03`,
        },
        options = {
            {
                type = "client",
                event = "qb-rental:openMenu",
                icon = "fas fa-car",
                label = "Rent Vehicle",
            },
        },
        distance = 4.0
    },
    
    so basically any ped model on the street that is spawned using "a_m_y_business_03 will function as the rental NPC

**Comment by ghostedxcx** [comment]
You could just change it to a ped that isn't used in the game world as i know there are some that where only used in cutscenes etc annoying as most do not fit the job style but.
